crates/router/src/connector/cashtocode/transformers.rs

```diff
@@ -0,0 +1,232 @@
+use common_utils::pii::Email;
+use masking::Secret;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    connector::utils::RouterData,
+    core::errors,
+    services,
+    types::{self, api, storage::enums},
+};
+
+//TODO: Fill the struct with respective fields
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
```

Comment:
Do we need Default, Eq, PartialEq?

Reply:
Default might be needed for tests. Removing Eq and partialEq